torch/_tensor_docs.py

```diff
@@ -1025,7 +1025,7 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('cross',
                r"""
-cross(other, dim=-1) -> Tensor
+cross(other, dim=None) -> Tensor
```

Comment:
Since the default is `None`, it will be good to revise the sample inputs function for cross, and have a negative dimension case. You can also think of more interesting cases (if any) for `cross`/`linalg.cross`.

Reply:
Done!